web-components/src/components/taskitem/TaskItem.ts

```diff
@@ -30,6 +29,43 @@ export namespace TaskItem {
     @property({ type: String }) customAriaLabel = "";
     @property({ type: String }) iconSrc = "";
     @property({ type: String }) tabIndexForContainer = "0";
+    @property({ type: String }) titleValue = "";
+    @property({ type: String }) itemTitleValue = "";
+
+    /**
+    * @deprecated Use `itemTitle` instead.
+    * Using title may cause an unwanted tooltip.
+    */
+    @property({ type: String, reflect: true})
+    get title(): string {
+      return this.displayTitle;
+    }
+    set title(title: string) {
+      this.titleValue = title;
+      this.updateTitle();
+    }
+    /**
+    * The visible text for the Title.
+    */
+    @property({ type: String, reflect: true, attribute: "itemtitle" })
+    get itemTitle() {
```

Comment:
update the attribute to item-title

Reply:
when this is item-title you can use the attribute item-title="title" or .itemTitle="title"